suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -675,9 +682,18 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
             return;
         }
 
-        // Start discovery for every network account type.
+        // Start discovery for every network - normal accounts as well as alternative accounts
         networksFiltered.forEach(network => {
```

Comment:
`createDescriptorPreloadedDiscoveryThunk` and `discoverNetworkBatchThunk` are still dispatched for all account types for example in case BTC or ADA is selected, because there might be unsupported account types: coinjoin (btc) or legacy (ada).

This is because now we consider all network undiscovered if those two are not. This makes discovery to be started even when not needed. This causes some unwanted consequences (possible passphrase requests, discovery and fiat rates loading, etc.)

Solution would probably be `selectNetworksWithUnfinishedDiscovery` renamed and returning map with not discovered networks with not discovered account types and exclude unsupported ones before starting discovery. I can make PR for this to merge here, so you dont have to bother with that. Lets discuss tomorrow on call or in DMs.